mods/Fifty.ServerUtilities/mod/scripts/vscripts/fsu_core.nut

```diff
@@ -52,12 +53,15 @@ void function FSU_PrintPrematchMessage() {
  * @param adminHeaderCode The ANSI code to be used for the admin header ( "[ADMIN]" )
  * @param ownerHeaderCode The ANSI code to be used for the owner prefix ( "[OWNER]" )
 */
-void function FSU_SetTheme( string headerCode, string highlightCode, string textCode, string adminHeaderCode, string ownerHeaderCode ) {
+void function FSU_SetTheme( string headerCode, string highlightCode, string textCode, string adminHeaderCode, string ownerHeaderCode, string errorCode, string successCode, string announceCode ) {

```

Comment:
I'll probably move this to ConVars as adding arguments to a function every update isn't backwards compatible.